release-notes/azure/v11.2.2.md

```diff
@@ -0,0 +1,12 @@
+⚡️ Giant Swarm Release 11.2.2 for Azure ⚡️
```

Comment:
Also add link to README like here: https://github.com/giantswarm/releases/pull/218/files

Reply:
will do that